### PR TITLE
Fix imports

### LINKED ISSUE #177: Custom Path Aliases Part 2
Following #167 

Replace the following direct path of `ducks, interface, components, utils, navigation, style`

So the following : 

``` 
import { emissions } from "../../../ducks";
import { Emission } from "../../../interfaces";
import { calculation, ui } from "../../../utils";
import { Text, Button, StickersImage } from "../../components";
import { Colors, ComponentsStyle } from "../../style";
import { navigate } from "../../navigation";
```

Should be transformed into : 

```
import { emissions } from "ducks";
import { Emission } from "interfaces";
import { calculation, ui } from "utils";
import { Text, Button, StickersImage } from "components";
import { Colors, ComponentsStyle } from "style";
import { navigate } from "navigation";
```

_NB1 : Will affect a lot of files but it's an easy task
NB2 : due to spam, I won't assigned this task to anyone without previous contribution to this repo but PR are accepted, just have a look if there is no PR up and if this issue is not assigned to anyone._

### PR DESCRIPTION
**Pull request by hb2708**
✅ I have read the [contributing file](https://github.com/NMF-earth/nmf-app/blob/master/contributing.md)

## Summary
This fixes #177

## Changelog
Updated all the imports as per ticket

### CODE REVIEW

**Comment by hb2708** [comment]
@PierreBresson Ready for review

Imports for interfaces 
But gave error in 
ui.ts, emissions.selectors.ts,
So skipped that.

File: app/components/SelectableListItem/SelectableListItem.tsx [hunk 1]
```diff
@@ -2,9 +2,10 @@ import React from "react";
 import { TouchableOpacity } from "react-native";
 import { Ionicons } from "@expo/vector-icons";
 
+import Text from "components/Text";
```
Comment (PierreBresson) [inline app/components/SelectableListItem/SelectableListItem.tsx:5]: import {Text} from "components";

File: app/navigation/Navigator/BottomTabNavigator.tsx [hunk 2]
```diff
@@ -2,10 +2,9 @@ import React from "react";
 import { createBottomTabNavigator } from "@react-navigation/bottom-tabs";
 import { useSafeAreaInsets } from "react-native-safe-area-context";
 
-import { Colors } from "../../style";
-
-import TabBarIcon from "../../components/TabBarIcon";
-import { t } from "../../utils";
+import TabBarIcon from "components/TabBarIcon";
```
Comment (PierreBresson) [inline app/navigation/Navigator/BottomTabNavigator.tsx:5]: import {TabBarIcon} from "components";

File: app/screens/ActDetail/ActDetailScreen.styles.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import { StyleSheet } from "react-native";
 
-import { Layout } from "../../style/layout";
+import { Layout } from "style/layout";
```
Comment (PierreBresson) [inline app/screens/ActDetail/ActDetailScreen.styles.ts:3]: from "style";

File: app/screens/AddEmission/AddEmissionScreen.tsx [hunk 4]
```diff
@@ -6,7 +6,19 @@ import { TransportEnum, FoodEnum, StreamingEnum } from "carbon-footprint";
 import DateTimePickerModal from "react-native-modal-datetime-picker";
 import { KeyboardAwareScrollView } from "react-native-keyboard-aware-scroll-view";
 
-import { Text, Tag, TextInput } from "../../components";
+import { Text, Tag, TextInput } from "components";
+import { userPreferences } from "ducks";
+import { EmissionType } from "interfaces";
+import { EmissionPayload } from "interfaces/emission/emission.interface";
```
Comment (PierreBresson) [inline app/screens/AddEmission/AddEmissionScreen.tsx:12]: `import { EmissionType, EmissionPayload } from "interfaces";`
To be able to do that you simply need to import 2 times in index.ts files inside `interface`

File: app/screens/AddEmission/components/AddEmissionButton/AddEmissionButton.tsx [hunk 5]
```diff
@@ -2,11 +2,12 @@ import React from "react";
 import { useDispatch } from "react-redux";
 import uuid from "uuid";
 
-import { Text, Button } from "../../../../components";
+import { Text, Button } from "components";
+import { emissions } from "ducks";
+import { EmissionPayload } from "interfaces/emission/emission.interface";
```
Comment (PierreBresson) [inline app/screens/AddEmission/components/AddEmissionButton/AddEmissionButton.tsx:7]: Same as previous comment

File: app/screens/Budget/ducks/__tests__/BudgetScreen.selectors.test.ts [hunk 6]
```diff
@@ -1,12 +1,11 @@
 import { FoodEnum, TransportEnum } from "carbon-footprint";
-
 import moment from "moment";
 
-import { selectors } from "../";
-import emissions from "../../../../ducks/emissions";
-import { Emission, EmissionType } from "../../../../interfaces";
+import emissions from "ducks/emissions";
```
Comment (PierreBresson) [inline app/screens/Budget/ducks/__tests__/BudgetScreen.selectors.test.ts:4]: import {emissions} from "ducks";

File: app/screens/Emissions/components/EmissionsList/__tests__/EmissionsList.test.tsx [hunk 7]
```diff
@@ -1,12 +1,12 @@
 import React from "react";
 import { create } from "react-test-renderer";
-
 import { FoodEnum, TransportEnum } from "carbon-footprint";
 
-import EmissionsList from "../EmissionsList";
+import emissions from "ducks/emissions";
```
Comment (PierreBresson) [inline app/screens/Emissions/components/EmissionsList/__tests__/EmissionsList.test.tsx:5]: import {emissions} from "ducks";

File: app/screens/Emissions/ducks/__tests__/EmissionsScreen.selectors.test.ts [hunk 8]
```diff
@@ -1,11 +1,11 @@
 import { FoodEnum, TransportEnum } from "carbon-footprint";
-
 import moment from "moment";
 
+import emissions from "ducks/emissions";
```
Comment (PierreBresson) [inline app/screens/Emissions/ducks/__tests__/EmissionsScreen.selectors.test.ts:4]: import {emissions} from "ducks";

File: app/redux/rootReducer.ts [hunk 9]
```diff
@@ -1,8 +1,8 @@
 import { combineReducers } from "@reduxjs/toolkit";
 
-import emissions from "../ducks/emissions";
-import budget from "../ducks/budget";
-import userPreferences from "../ducks/userPreferences";
+import emissions from "ducks/emissions";
```
Comment (PierreBresson) [inline app/redux/rootReducer.ts:3]: import {emissions,budget,userPreferences} from "ducks";

File: app/screens/Budget/components/NumberOfDaysVegetarian/ducks/__tests__/NumberOfDaysVegetarian.selectors.test.ts [hunk 10]
```diff
@@ -1,10 +1,10 @@
 import { FoodEnum } from "carbon-footprint";
-
 import moment from "moment";
 
+import emissions from "ducks/emissions";
```
Comment (PierreBresson) [inline app/screens/Budget/components/NumberOfDaysVegetarian/ducks/__tests__/NumberOfDaysVegetarian.selectors.test.ts:4]: import {emissions} from "ducks";

**Review by PierreBresson — CHANGES_REQUESTED** [review]
Very simples changes, otherwise looks good.
I've put back the `hacktoberfest`, I thought it was over... Timezone issue, sorry about that!

**Comment by hb2708** [comment]
@PierreBresson thanks for review will fix the changes tonight. And also thanks for adding the tag back. But I think it will not count anymore as i was late to create pr. They are just counting 3 pr.

**Comment by hb2708** [comment]
@PierreBresson All should be fixed now  Give a look thanks.

**Review by PierreBresson — APPROVED** [review]
🏄‍♂️

**Comment by PierreBresson** [comment]
@hb2708 thanks a lot !

**Pull request merged**